coordinator/src/node.rs

```diff
@@ -208,17 +192,45 @@ enum TradeAction {
     // Reduce,
 }
 
+/// Calculates the accept settlement amount based on the pnl.
+///
+/// The smallest possible result is 0.
```

Comment:
The type already encodes this information 😛

Reply:
fair enough 🙈